src/client/ducks/pullrequests/actions/index.js

```diff
@@ -1,10 +1,11 @@
 /* @flow */
 
 import _ from 'lodash'
-import type { PullRequestSource } from 'universal/types'
+import type { PullRequestSourceType } from 'universal/types'
+import { PullRequestSource } from 'universal/constants'
 import type { OrderByType } from 'ducks/order'
 import { fetchOnoActionCreator, fetchOnoAction } from 'ducks/fetch'
-import type { FetchAction } from 'ducks/fetch'
+import type { FetchAction, FetchPullRequestVariablesType } from 'ducks/fetch'
```

Comment:
We probably would put this type here, as `fetch` is more generic, and it would be easy for us to see what inside and edit it here if necessary.